src/morse/index.js

```diff
@@ -58,6 +58,27 @@ const MORSE_CODE = {
 
 Object.freeze(MORSE_CODE);
 
-function morse(text) {}
+function morse(text) {
+  let ans = "";
+  if (typeof text != "string") {
+    throw "Please provide a morse string";
+  } else if (text == "") {
```

Comment:
You don't need this else since the throw would exit the program. This is a separate check here too.